### DISCUSSION TITLE
Desk tool crashes with 'Error: Minified React error #130' in built v3 studio

### DISCUSSION THREAD

**Discussion by ilyagru** [sanity-io/sanity#3715] [Sanity Studio Support]
Hi!

[V3 embedded into a next.js app]

In dev environment all works as expected but after I build the app and try to open the desk tool and go to Post, it crashes with this error `Error: Minified React error #130; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=boolean&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.`
However, if I open the editor for another simpler model called ‘Information’, it opens the editor fine.

Any idea what could go wrong?

<img width="300" alt="Screenshot 2022-10-02 at 11 46 39" src="https://user-images.githubusercontent.com/18074511/193776521-15c27cf8-7f5f-4025-b11d-fb5b2b5bf043.png">
<img width="300" alt="Screenshot 2022-10-02 at 11 47 38" src="https://user-images.githubusercontent.com/18074511/193776515-36bf0b72-52ce-4e84-af4a-1b0024636571.png">

**Comment by lourencelinao** [comment]
Hello  @ilyagru , I am currently experiencing this issue, have you found a fix for this?

**Reply by ilyagru** [reply]
Hey, please take a look at this issue. https://github.com/sanity-io/sanity/issues/3708
